db/l2db/l2db.go

```diff
@@ -267,20 +267,22 @@ func (l2db *L2DB) addTxs(txs []common.PoolL2Tx, checkPoolIsFull bool) error {
 	}
 	// Query begins with the insert statement
 	query := queryInsertPart
+	const onConflictQuery = ` ON CONFLICT ON CONSTRAINT tx_id_unique DO UPDATE SET
+			from_idx = excluded.from_idx, to_idx = excluded.to_idx, to_eth_addr = excluded.to_eth_addr, to_bjj = excluded.to_bjj,
+			token_id = excluded.token_id, amount = excluded.amount, fee = excluded.fee, nonce = excluded.nonce, state = excluded.state,
+			info = excluded.info, signature = excluded.signature, rq_from_idx = excluded.rq_from_idx, rq_to_idx = excluded.rq_to_idx,
+			rq_to_bjj = excluded.rq_to_bjj, rq_token_id = excluded.rq_token_id, rq_amount = excluded.rq_amount, rq_fee = excluded.rq_fee, rq_nonce = excluded.rq_nonce,
+			tx_type = excluded.tx_type, amount_f = excluded.amount_f, client_ip = excluded.client_ip,
+			rq_offset = excluded.rq_offset, max_num_batch = excluded.max_num_batch WHERE tx_pool.atomic_group_id IS NULL`
+
 	if checkPoolIsFull {
```

Comment:
As this is very critical to the hermez-node I agree with @arnaubennassar and I would recommend to add tests to this item